lib/connextclient/ConnextClient.ts

```diff
@@ -50,13 +51,15 @@ interface ConnextClient {
   on(event: 'htlcAccepted', listener: (rHash: string, amount: number, currency: string) => void): this;
   on(event: 'connectionVerified', listener: (swapClientInfo: SwapClientInfo) => void): this;
   on(event: 'depositConfirmed', listener: (hash: string) => void): this;
+  on(event: 'lowTradingBalance', listener: (alert: BalanceAlert) => void): this;
```

Comment:
This can use the new `Alert` type, I described above, same for other places the `lowTradingBalance` event is defined.